scripts/reference-tables.template

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <meta content="width=device-width, initial-scale=1.0" name="viewport" />
+  <title>Index of ARIA Design Pattern Examples | WAI-ARIA Authoring Practices 1.1</title>
+  <link rel="stylesheet" href="https://www.w3.org/StyleSheets/TR/2016/base.css">
+  <link rel="stylesheet" href="css/core.css">
+  <script src="js/examples.js"></script>
+  <script src="js/highlight.pack.js"></script>
+  <script src="js/app.js"></script>
+</head>
+<body>
+  <nav aria-label="ARIA Practices">
```

Comment:
I changed aria-label here and in the footer. It was previously "Top link to ARIA Practices". Per our landmark guidance, if two nav sections have identical content and identical purpose, they should have the same label, not distinct labels. Distinguishing regions with positional terms, "top" and "bottom" is not recommended.